app/assets/stylesheets/response_view.sass

```diff
@@ -27,6 +34,17 @@
     .fa-warning
       color: darken($screensmart-red, 2%)
       padding-right: 0.67rem
+  .results-table
+    border: 2px solid $response-estimate-interpretation-border-color
+    background-color: $response-background-color
+    padding: 0.5rem 1.6rem
+    border-radius: $response-border-radius
```

Comment:
Prima om die variabelen te hergebruiken, maar nu is het geen `response-border-radius` meer, maar bijvoorbeeld een `block-border-radius` (zelfde voor background-color etc). Wellicht even hernoemen?